zone/fearpath.cpp

```diff
@@ -157,41 +157,19 @@ void Mob::ProcessFlee()
 
 void Mob::CalculateNewFearpoint() {
 	if (RuleB(Pathing, Fear) && zone->pathing) {
-		auto Node = zone->pathing->GetRandomLocation();
+		auto Node = zone->pathing->GetRandomLocation(glm::vec3(GetX(), GetY(), GetZ()));
```

Comment:
Do you know if this selects valid locations? We were going to eventually look into this to see if it is at least grabbing random points inside of geometry or on tiles

Reply:
Yes - I rewrote the function to get the closest point near the closest geometry near a location.
I've tested it in hellish indoor scenarios (gukbottom near evil eye) and open fields (northkarana) and it works great.

Reply:
Sweet! I'm excited to see this